airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -79,13 +79,17 @@ data class Meta(
     }
 }
 
-data class DestinationRecord(
+sealed interface DestinationRecordPayload : DestinationRecordDomainMessage {
+    val serialized: String
+}
+
+data class DestinationRecordMarshaled(
```

Comment:
Should we rename this class?

Reply:
Renaming to `DestinationRecordAirbyteValue`